templates/zerver/help/import-from-gitter.md

```diff
@@ -1,105 +1,126 @@
-`<subdomain>` with the desired subdomain.
+{!import-self-hosted-server-tips.md!}
 
-```
-./manage.py import <subdomain> converted_gitter_data
-```
+    ```
+    cd /home/zulip/deployments/current
+    ./scripts/stop-server
+    ./manage.py convert_gitter_data /tmp/gitter_data.json --output /tmp/converted_gitter_data
+    ./manage.py import '' /tmp/converted_gitter_data
+    ./scripts/start-server
+    ```
 
-{!import-login.md!}
+    Alternatively, to import into a custom subdomain, run:
 
-## Create organization owners
+    ```
+    cd /home/zulip/deployments/current
+    ./scripts/stop-server
+    ./manage.py convert_gitter_data /tmp/gitter_data.json --output /tmp/converted_gitter_data
+    ./manage.py import <subdomain> /tmp/converted_gitter_data
+    ./scripts/start-server
+    ```
 
-The [Gitter API][gitter-api-user-data] don't contain data on which
-users are administrators of the Gitter channel.  As a result, all
-Gitter users are imported into Zulip as normal users.  You can follow
-the Zulip documentation on
-[making a user an organization owner from the terminal][grant-admin-access]
-to mark the appropriate users as organization owners.
+1. Follow [step 4](https://zulip.readthedocs.io/en/stable/production/install.html#step-4-configure-and-use)
+   of the guide for [installing a new Zulip
+   server](https://zulip.readthedocs.io/en/stable/production/install.html).
 
-[grant-admin-access]: https://zulip.readthedocs.io/en/latest/production/management-commands.html#grant-administrator-access)
-[gitter-api-user-data]: https://developer.gitter.im/docs/user-resource
+{end_tabs}
+
+#### Import details
 
-## Caveats
+Whether you are using Zulip Cloud or self-hosting Zulip, here are a few notes to
+keep in mind about the import process:
 
-- The [Gitter data export tool](https://github.com/minrk/archive-gitter)
-  doesn't support exporting private Gitter channels.
+- [Gitter's export tool](https://github.com/minrk/archive-gitter) does not export
+  workspace settings, so you will need to [configure the settings for your Zulip
+  organization](/help/customize-organization-settings). This includes settings
+  like [email visibility](/help/restrict-visibility-of-email-addresses),
+  [message editing permissions](/help/configure-message-editing-and-deletion#configure-message-editing-and-deletion_1),
+  and [how users can join your organization](/help/restrict-account-creation).
+
+- Gitter's export tool does not export user settings, so users in your organization
+  may want to [customize their account settings](/help/getting-started-with-zulip).
+
+- The [Gitter API][gitter-api-user-data] doesn't contain data on which
+  users are administrators of a Gitter channel.  As a result, all
+  Gitter users are imported into Zulip as normal users.  You can follow
+  the Zulip documentation on
+  [making a user an organization owner from the terminal][grant-admin-access]
+  to mark the appropriate users as organization owners.
```

Comment:
Should we mention that in the Zulip Cloud case they should let us know which account gets the org owner status? Or do we keep that silent to not be overly verbose in the paragraph and we can explain that to them during the actual email conversation when they message support to do the import?

Reply:
Yeah, let's include it in the instructions to avoid unnecessary back-and-forth.